install.sh

```diff
@@ -9,9 +9,8 @@ apt-get install -y libxml2 libxml2-dev libxslt1.1 libxslt1-dev libffi-dev libssl
 # Installing pip-10.0.1
 wget -nv https://bootstrap.pypa.io/3.2/get-pip.py
 python get-pip.py
-python -m pip install --upgrade pip
-
-pip install -U pip setuptools
+python -m pip install pip==1.5.4
```

Comment:
why is this needed? isnt the default version already 1.5?

Reply:
ya, default version is 1.5.4

Reply:
then why is this required?

Reply:
ok, i will update